tool/state-processor/src/system/mod.rs

```diff
@@ -237,6 +237,11 @@ where
 			kton_details,
 		);
 
+		// Transfer RING from special accounts to the treasury account.
+		//
+		// It has been confirmed that these accounts do not have KTON.
+		S::SPECIAL_ACCOUNTS.iter().for_each(|a| self.shell_state.transfer_all(a, S::TREASURY));
```

Comment:
This pr focuses handling all special account's without KTON asset, we still need some migration or democracy process to deal with KTON after 2.0 set up, right?

Reply:
I mean that there are some other accounts with KTON asset needs migration or force transfer.

Reply:
No more special accounts.